cmd/virt-handler/virt-handler.go

```diff
@@ -346,6 +347,7 @@ func (app *virtHandlerApp) Run() {
 		downwardMetricsManager,
 		capabilities,
 		hostCpuModel,
+		netbinding.NewMemoryOverheadCalculator(app.clusterConfig),
```

Comment:
I agree it is better to do it from here, but it is now inconsistent with the `netconf` and `netstat`.
Is this a must or can be move all network related stuff to the `cmd` level as a follow up?

Reply:
I prefer to keep it as-is in order for `pkg/virt-handler/vm.go` to be decoupled from `kubevirt.io/kubevirt/pkg/network/netbinding`.

We could do the same with `netconf` and `netstat` in a follow-up.